apps/harpguru/App.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import { View, TouchableOpacity, Image } from 'react-native';
+import { View, TouchableOpacity, Text, Image } from 'react-native';
```

Comment:
There's a warning here because `Image` is unused.

I'm going to leave that in since the next project is to get typechecking etc working across packages with lerna.